R/audit_deps.R

```diff
@@ -16,7 +16,10 @@
 get_pkgs = function(pkgs = NULL) {
   if (is.null(pkgs)) {
     cli::cli_alert_info("Calling {.pkg installed.packages()}, this may take time")
+    # excluding base and recommended packages
```

Comment:
Actually there is no good reason to skip base and recommended. Would you fix?

Reply:
I THINK I skipped these because iirc they were included with an R install, and thus not really "third party", or stuff you chose to install. That was at least my thinking. Y'all can switch it to everything if you think that's good!

Reply:
@DarthHater That is correct and that makes sense. But still good to give 'em a check :)

Reply:
Zero objections to checking them, up to y'all (you are obviously way better R experts than I am :) )